speech/google/cloud/speech/result.py

```diff
@@ -46,9 +93,28 @@ def from_pb(cls, response):
         :rtype: :class:`~google.cloud.speech.result.StreamingSpeechResult`
         :returns: Instance of ``StreamingSpeechResult``.
         """
-        alternatives = [Alternative.from_pb(alternative)
-                        for alternative in response.alternatives]
+        alternatives = []
```

Comment:
I did that to get around [this bug](https://bitbucket.org/ned/coveragepy/issues/515/coverage-of-multiline-generator-set-dict).